crates/vhost-user-backend/CHANGELOG.md

```diff
@@ -9,10 +9,23 @@
 
 ### Deprecated
 
+## v0.9.0
+
+### Added
+- [[#138]](https://github.com/rust-vmm/vhost/pull/138): vhost-user-backend: add repository metadata
+
+### Changed
+- Updated dependency virtio-bindings 0.1.0 -> 0.2.0
+- Updated dependency virtio-queue 0.7.0 -> 0.8.0
+- Updated dependency vm-memory 0.10.0 -> 0.11.0
+
+### Fixed
+- [[#154]](https://github.com/rust-vmm/vhost/pull/154): Fix return value of GET_VRING_BASE message
```

Comment:
We should add also
```
-  [[#142]](https://github.com/rust-vmm/vhost/pull/142): vhost_user: Slave requests aren't only FS specific
```

Since it covered both vhost and vhost-user-backend, maybe we need to add the same line also in the vhost's changelog

Reply:
You're right, done.